### DISCUSSION TITLE
Disabling / pusing hot reload in a folder?

### DISCUSSION THREAD

**Discussion by zutamacka** [quasarframework/quasar#13125] [CLI - Electron mode]
Hi all, 

I'm writing an app (q-sandbox) that allows the user to upload pdf files (single or batch). Files are saved to **q-sandbox\public\pdfjs\pdfs** in order to allow easy access for the pdfjs viewer. 

However, every time my backend script saves a single file there, my app hot reloads and the $q.loading, $q.dialog features I wrote get interrupted and the app appears as if the backend script has finished. But it didn't, it can in fact take minutes and minutes (files get additionally processed). 

Is there a way to exclude the q-sandbox\public\pdfjs\ folder from hot reload? Or do I needn't bother because it doesn't happen in production? 

Thankful for any help :) 

PS.
I'm sorry if this is basic, I am a n00b

**Comment by ontwikkelfabriek** [comment]
@zutamacka 

Depends  if you are using Vite or Webpack for managing your project.

Here's info on how to exlude a folder with Webpack( quasar.conf.js):
https://stackoverflow.com/questions/42085609/ignoring-files-with-webpack-dev-server

Webpack docs:
https://webpack.js.org/configuration/watch/

**Reply by zutamacka** [reply]
Thank you so much!!!

Edit: 
For other n00bs like me who stumble upon this; below is what I did to get it to work. 
It required checking out version changes in webpack etc
https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md

quasar.conf.js : 

```
    // Full list of options: https://quasar.dev/quasar-cli/quasar-conf-js#Property%3A-devServer
    devServer: {
      server: {
        type: 'http'
      },
      port: 8080,
      open: true, // opens browser window automatically
      static: {
        watch: {
          ignored: [
            path.resolve(__dirname, 'public\\pdfjs')
          ]
      }
      }
    }
```
